tests/apps/test_morph_stats.py

```diff
@@ -68,14 +68,15 @@
     'neurite_type': ['AXON', 'APICAL_DENDRITE', 'BASAL_DENDRITE', 'ALL'],
     'morphology': {
         'soma_radius': {'modes': ['mean']},
-        'max_radial_distance': {'modes': ['mean']},
+        'max_radial_distance': {'modes': ['mean', 'raw']},
     }
 }
 
 REF_OUT = {
     'morphology': {
         'mean_soma_radius': 0.13065629648763766,
         'mean_max_radial_distance': 99.5894610648815,
+        'raw_max_radial_distance': 99.5894610648815,
```

Comment:
so, `raw_max_radial_distance` and `mean_max_radial_distance` are the same? is it ok?

Reply:
there is just one number here

Reply:
we can put raw in another entry if we want, but it may yield many numbers